data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/GeoIPProcessor.java

```diff
@@ -40,6 +43,7 @@ public class GeoIPProcessor extends AbstractProcessor<Record<Event>, Record<Even
   static final String GEO_IP_EVENTS_PROCESSED = "eventsProcessed";
   static final String GEO_IP_EVENTS_FAILED_LOOKUP = "eventsFailedLookup";
   static final String GEO_IP_EVENTS_FAILED_ENGINE_EXCEPTION = "eventsFailedEngineException";
+  static final List<String> DATABASE_EXPIRED_TAGS = List.of("database_expired");
```

Comment:
Also, I think we are looking toward a convention of adding default tags with a leading underscore. See:

https://github.com/opensearch-project/data-prepper/pull/4118/files#diff-82214defcb3cec276e04f9c58345825005c3e1a8ef444827e85985ddc84f191fR31